install.sh

```diff
@@ -11,6 +11,8 @@ VERSION="$1"
 ARCH=""
 # The tag search term to use if no version is specified (first match is used)
 SEARCH_TERM="cadence-v1.0.0"
+# The specific version to download as "flow"
+FLOW_VERSION="v1.20.5"
```

Comment:
Same as above for the PowerShell script above, maybe avoid hard-coding a specific release

Reply:
Improved this in [0fa235d](https://github.com/onflow/flow-cli/pull/1663/commits/0fa235da235ee62241ce58955bfe74130c33f72e)